### ISSUE TITLE
Missing Hue integration - smart bulbs can not be controlled from Alexa devices

### ISSUE THREAD

**Issue by MnM001** [openshwprojects/OpenBK7231T_App#859]
Hi,

It seems that at this point in time only WEMO integration is enabled. Alexa finds my smart lights as switches. I can only turn them on or off from Alexa. I cant control color, dimming etc.

Will be good to have Hue integration working so Alexa can correctly identify smart lights as lights and be controlled accordingly by Alexa devices.

Any plans or ETA for Hue integration?

Cheers.

**Comment by openshwprojects** [comment]
Yes, it's on TODO list, and btw, we can need some testers for that, so please be around for the time being and maybe you can help if you want.

**Comment by MnM001** [comment]
Hi,

Sounds good that this is on the radar.
Happy to help with testing. Just let me know when.
Cheers.

**Comment by CLAM01** [comment]
I just saw this feature request.
This is the same topic as #885.

I am also happy to test LSC Mood Light from Action Markt Netherlands.

https://www.action.com/de-de/p/lsc-smart-connect-stimmungslampe/

If this feature works, then I could control everything via Alexa in my home.

**Comment by CLAM01** [comment]
Hi, is there already a roadmap or milestone planning for when the driver will be finalized?

**Comment by openshwprojects** [comment]
I will try to look into it again soon, as said on forum. I am trying to solve issues one by one but I am just one person. Do you know C and would like to help?

**Comment by CLAM01** [comment]
Hi, unfortunately I don't know C or any other programming language, otherwise I would love to support.

**Comment by Milinator** [comment]
@openshwprojects I would like to help to add Hue Emulation Support. I would even pay money to make it work.
